### PR TITLE
MGMT-8564: Rename the UI deployment template

### PR DESCRIPTION
**Pull request by eliorerz**
# Assisted Pull Request

## Description
Fix failing CI due to a template rename on  https://github.com/openshift-assisted/assisted-ui/commit/1f499bfc5d94286039d99da3e7030fe6cb847884

## List all the issues related to this PR

- [ ] New Feature
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @osherdp

### CODE REVIEW

**Review by osherdp — APPROVED** [review]

**Review by osherdp — APPROVED** [review]

**Review by osherdp — APPROVED** [review]

File: tools/deployment_options.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def get_tag(image_fqdn):
     return image_fqdn.split(":")[-1]
 
 
-def get_image_override(deployment_options, short_image_name, env_var_name):
+def get_image_override(deployment_options, short_image_name, env_var_name, repo="ocpmetal"):
```
Comment (lranjbar) [inline tools/deployment_options.py:103]: Nit but the term for this is "org" instead of "repo"

For images the pattern is `<registry>/<org>/<repo>:<tag>`

**Review by osherdp — APPROVED** [review]

**Pull request merged**